docs/DOCKER.md

```diff
@@ -3,14 +3,11 @@
 Docker can be used to setup a local development environment:
 
 - Ensure Docker is installed ([Docker Desktop](https://www.docker.com/products/docker-desktop) is a good option for developers)
-- From the root of this project, run `docker-compose up -d`
-- Once <http://localhost:8082> displays the WordPress install screen, run `./bin/docker-setup.sh`
+- From the root of this project, run `npm run up`
 - The fully configured site can now be accessed on <http://localhost:8082>
 - The prompt to run the setup wizard can be dismissed unless there is something specific you would like to configure
```

Comment:
> - The prompt to run the setup wizard can be dismissed unless there is something specific you would like to configure

I believe this step is not valid anymore.

Reply:
After setup, going to the WooCommerce menu option will show the wizard.
I also updated the setup script to the latest version used by WCPay... and the skip is still necessary

Reply:
Ok 👍  I was interpreting this setup wizard as the WordPress install wizard, which appears in the previous step, before running `./bin/docker-setup.sh` in the previous documentation.